docs/cookbook/mockery_on.rst

```diff
@@ -11,7 +11,7 @@ when called. If the closure returns ``true``, Mockery will consider that the
 argument has passed the expectation. If the closure returns ``false``, or a
 "falsey" value, the expectation will not pass.
 
-The ``\Mockery::on()`` matcher can be used in various scenarios &mdash; validating
+The ``\Mockery::on()`` matcher can be used in various scenarios — validating
```

Comment:
It seems like the HTML entity is escaped in the resulting docs: http://docs.mockery.io/en/latest/cookbook/mockery_on.html